### PR TITLE
Issue 2 add project structure attempt 2

### PR DESCRIPTION
**Pull request by ZoltanOnody**

### CODE REVIEW

**Comment by ZoltanOnody** [comment]
@viktorria teba neviem assignut lebo niesi v projekte a nic si tu este nespravila... kazdopadne prosim skuste si to vsetci rozbehat podla navodu,... ak vam to pojde dajte approve, pokial nie, pytajte sa. mergnut to mergnem potom ja.

File: README.md [hunk 1]
```diff
@@ -1 +1,64 @@
-# Nobelisti
\ No newline at end of file
+# Nobelisti
+
+## Instalation
+
+0. Ensure you have installed (docker)[https://docs.docker.com/engine/installation/] and (docker-compose)[https://docs.docker.com/compose/install/]
```
Comment (nitrajka) [inline README.md]: Links don't work. Can you change the brackets and parenthesis?

**Review by nitrajka — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1 +1,64 @@
...
+```
+
+2. Build and run the container
+```
+docker-compose up --build
+```
+
+3. Get docker's IP and save it to /etc/hosts
+```
+echo $(docker network inspect bridge | grep Gateway | grep -o -E '[0-9\.]+')
```
Comment (nitrajka) [inline README.md]: `docker-machine ip default` for OSX

File: README.md [hunk 3]
```diff
@@ -1 +1,64 @@
...
+```
+
+Open `/etc/hosts` and append `IP.ADDRESS.FROM.ECHO    symfony.dev`.
+
+
+4. Change permissions so symfony can save cache, logs & sessions.
+```
+sudo chmod -R 777 src/var/cache src/var/logs src/var/sessions
```
Comment (nitrajka) [inline README.md]: does not work on OSX

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,47 @@
+version: '2'
+
+services:
+    db:
```
Comment (nitrajka) [inline docker-compose.yml]: `user: "1000:50"` is needed to create a DB on OSX

File: README.md (hunk 3 above)
Comment (ZoltanOnody) [reply README.md]: I'll try to find a better solution for this.

File: docker-compose.yml (hunk 4 above)
Comment (ZoltanOnody) [reply docker-compose.yml]: done.

**Review by nitrajka — APPROVED** [review]

**Pull request merged**